models/ironbank_v2/goerli/ironbank_v2_goerli_markets.sql

```diff
@@ -0,0 +1,39 @@
+{{ config(
+    alias = 'markets',
+    post_hook='{{ expose_spells(\'["goerli"]\',
+                                "project",
+                                "ironbank_v2_markets",
+                                \'["michael-ironbank-v2"]\') }}'
+) }}
+
+SELECT
+    market, 
+    market_symbol,
+    market_decimals,
+    debt_token,
+    debt_token_symbol,
+    debt_token_decimals,
+    ib_token,
+    ib_token_symbol,
+    ib_token_decimals
+FROM
+    (
+        VALUES
+            ('0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6', 'WETH', 18, '0x66F373c3C8596Dd664902f0B237A626791351C32', 'debtWETH', 18, '0xE10AaAb9293074a506884d11FBEAf3026467db0a', 'ibWETH', 18),
```

Comment:
This looks a lot like a Spark Query -- no longer accepted in the spellbook. To contribute new spells, please use DuneSQL.

here are the instructions.

https://github.com/duneanalytics/spellbook/#new-spells